shogun-lib/src/main/java/de/terrestris/shogun/lib/model/jsonb/application/DefaultCrsDefinition.java

```diff
@@ -20,15 +20,13 @@
 import io.swagger.v3.oas.annotations.media.Schema;
 import lombok.AllArgsConstructor;
 import lombok.Data;
-import lombok.EqualsAndHashCode;
-import lombok.ToString;
+import lombok.NoArgsConstructor;
 
 import java.io.Serializable;
 
 @Data
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@ToString
-@EqualsAndHashCode
+@NoArgsConstructor
```

Comment:
`force=true` required here?

Reply:
This is needed for classes for fields annotated with `NonNull` only.